wa_examples/html_wa.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "/wa/translation-alignment/QWxpZ25tZW50Q2h1bmtOb2RlOjE=",
+  "type": "Annotation",
+  "target": [
+    "urn:cts:greekLit:tlg0012.tlg001.msA:1.1-1.7",
+    {
+      "type": "SpecificResource",
+      "source": {
+        "id": "https://rosetest.library.jhu.edu/rosademo/iiif/homer/VA/VA012RN-0013/canvas",
+        "type": "Canvas"
+      },
+      "selector": {
+        "type": "ImageApiSelector",
+        "region": "pct:6.11,22.52,46.75,15.01"
+      }
+    }
+  ],
+  "body": [
```

Comment:
I chose two body entries, with each entry identifying the language.

We could also have a single body with HTML representing _both_ sides of the alignment (similar to https://gist.github.com/jacobwegner/eb3de155d9d7a62cdf4c48838cecb7ed#file-html_body-json)

Reply:
I think this is a good start. Let's see how it looks slapped into Mirador 3.